spec/models/api_response_spec.rb

```diff
@@ -0,0 +1,133 @@
+require "rails_helper"
+require "fake_issues"
+
+RSpec.describe ApiResponse do
+  context "a successful response" do
+    let(:params) { Net::HTTPSuccess.new(1.1, "200", "OK") }
+    let(:response_body) do
+      {
+        data: {
+          grantee: "Some Grantee"
+        },
+        meta: {
+          itemTotalCount: 20
+        }
+      }
+    end
+    let(:subject) { ApiResponse.new(params) }
```

Comment:
[suggestion] I wonder if you would consider renaming `params` here? I was confused about why params would be getting passed to the ApiResponse because I was conflating the term with controller params. I think I would be less confused if this was `response` or `res` or `http_obj` or many other terms!